lib/util/collection.js

```diff
@@ -96,6 +96,17 @@ Collection.prototype.nextPage = function() {
   }
 };
 
+/**
+ * Checks if Collection can fetch next page
+ * @returns {boolean}
+ */
+Collection.prototype.isPaginateable  = function () {
```

Comment:
Not sure if isPaginateable is the best name, but maybe somebody else can chip in.
There are a bunch of extra spaces here that should be cleaned up.

Also, can you provide more detail on the endpoints that don't have pagination available and the error you're getting?
Sample commands will be helpful. I've been trying some endpoints but I either get an invalid request or pagination keeps kicking in even if I try to override it. Thanks!